assets/configs/tenancy.php

```diff
@@ -30,6 +30,17 @@
         // Must implement \Hyn\Tenancy\Contracts\Website
         'website' => \Hyn\Tenancy\Models\Website::class
     ],
+    /**
+     * The package middlewares. Removing a middleware here will disable it.
+     * You can of course extend/replace them or add your own.
+     */
+    'middlewares' => [
```

Comment:
One comment.  `middlewares` sounds weird to me (native-born English speaker).  Much like `deer`, `fish`, and even `software`, the pluralization of `middleware` is often just `middleware`.  Also, for reference: https://english.stackexchange.com/questions/257120/middleware-vs-middlewares

Reply:
That’s a good point. I’m not a native English speaker.

What would you suggest to do in a foreach statement (ex. right now it’s foreach (middlewares as middleware)) to clearly distinguish the plural from the singular?